demos/rerank/README.md

```diff
@@ -16,142 +16,85 @@ It will create an image called `openvino/model_server:latest`.
-        },
-        {
-            "config": {
-                "name": "rerank_model",
-                "base_path": "/workspace/models/BAAI/bge-reranker-large-rerank"
-            }
-        }
-    ],
-    "mediapipe_config_list": [
-        {
-            "name": "rerank",
-            "graph_path": "/workspace/models/graph.pbtxt"
-        }
-    ]
-}
+tree models
+models
+├── BAAI
+│   └── bge-reranker-large
+│       ├── graph.pbtxt
+│       ├── rerank
+│       │   └── 1
+│       │       ├── model.bin
+│       │       └── model.xml
+│       ├── subconfig.json
+│       └── tokenizer
+│           └── 1
+│               ├── model.bin
+│               └── model.xml
+└── config.json
 
 ```
+> **Note** The actual models support version management and can be automatically swapped to newer version when new model is uploaded in newer version folder.
+
 
+## Deployment 
 
-## Start-up
 ```bash
-docker run -d --rm -p 8000:8000 -v $(pwd)/:/workspace:ro openvino/model_server:latest --port 9000 --rest_port 8000 --config_path /workspace/config.json
+docker run -d --rm -p 8000:8000 -v $(pwd)/models:/workspace:ro openvino/model_server:latest --port 9000 --rest_port 8000 --config_path /workspace/config.json
 ```
 
 ## Client code
 
 
-```python
-import requests
-
-# Define the API endpoint and your API key
-api_endpoint = "http://localhost:8000/v3/rerank"
-api_key = "YOUR_API_KEY"
-
-# Create the payload
-payload = {
-    "model": "rerank",
-    "query": "what is it?",
-    "documents": [
-        "one text",
-        "two text longer than one"
-    ]
-}
-
-# Set up the headers
-headers = {
-    "Authorization": f"Bearer {api_key}",
-    "Content-Type": "application/json"
+```bash
+curl http://localhost:8000/v3/rerank  -H "Content-Type: application/json" \
+-d '{ "model": "BAAI/bge-reranker-large", "query": "welcome", "documents":["good morning","farewell"]}' | jq .
+```
+```json
+{
+  "results": [
+    {
+      "index": 0,
+      "relevance_score": 0.3886180520057678
+    },
+    {
+      "index": 1,
+      "relevance_score": 0.0055549247190356255
+    }
+  ]
 }
-
-# Make the POST request
-response = requests.post(api_endpoint, json=payload, headers=headers)
-
-# Handle the response
-if response.status_code == 200:
-    reranked_results = response.json()
-    print("Reranked Results:", reranked_results)
-else:
-    print("Error:", response.status_code, response.text)
-
 ```
 
-Check OVMS logs for results (serialization is not implemented yet)
-
-```
--0.64591
--6.86708
-0.343912 0.00104043 
-```
 
-## Comparison with LangChain OpenVINOReranker
+## Comparison with Hugging Faces
 
-```python
-from langchain_community.document_compressors.openvino_rerank import OpenVINOReranker, RerankRequest
+```bash
+python compare_results.py --query "hello" --document "welcome" --document "farewell"
+query hello
+documents ['welcome', 'farewell']
+HF Duration: 145.731 ms
+OVMS Duration: 23.227 ms
+HF reranking: [0.99640983 0.08154089]
+OVMS reranking: [0.9968273 0.0913821]
```

Comment:
Why does it differ? It was exactly the same during my tests. Isn't it different model compression?

Reply:
yes, this is int8 quantized model

Reply:
Shouldn't we showcase that accurracy is 100%?